page_forms/encrypted_page.py

```diff
@@ -0,0 +1,20 @@
+from selenium.webdriver.common.by import By
+from page_forms.base_page import BasePage
+
+
+class EncryptedLink(BasePage):
+    FIRST_NAME = (By.XPATH, "//*[contains(@name, 'first_name')]")
+    LAST_NAME = (By.XPATH, "//*[contains(@name, 'last_name')]")
+    CITY_NAME = (By.XPATH, "//*[contains(@class, 'form-control city')]")
+    COUNTRY_NAME = (By.XPATH, "//*[contains(@id, 'country')]")
+    SUBMIT_BUTTON = (By.XPATH, "//*[contains(@class, 'btn btn-default')]")
+
+    def __init__(self, browser, url):
+        super().__init__(browser, url)
+
+    def fill_form(self):
```

Comment:
all parameters should be passed as function arguments. Please do not hardcode test data in the function, e.g.
def fill_login_form(first_name, last_name, city, country)